### DISCUSSION TITLE
Recommended Proxy Service

### DISCUSSION THREAD

**Discussion by ravenrd** [scholarly-python-package/scholarly#263] [Q&A]
Hi!

I am very much a fun of Scholarly, but I keep getting blocked by Google.
It is great that there is built in support for a number of proxy services, but which ones are actually working well in your experience? I am using FreeProxies, because it is the easiest one to set up and use, but it fails most of the time and I spend a lot of time in loops trying to get another one. Quite often, I cannot download all publications and citations from a single author in reasonable time. 
Do you have a recommendation for which proxy solution to use to get reliable results?

**Comment by ipeirotis** [comment]
So far, Luminati seems to work well. There is a PR for another proxy
service as well, which we plan to incorporate in future versions.

On Sun, Feb 21, 2021 at 02:53 Benny Akesson <notifications@github.com>
wrote:

> Hi!
>
> I am very much a fun of Scholarly, but I keep getting blocked by Google.
> It is great that there is built in support for a number of proxy services,
> but which ones are actually working well in your experience? I am using
> FreeProxies, because it is the easiest one to set up and use, but it fails
> most of the time and I spend a lot of time in loops trying to get another
> one. Quite often, I cannot download all publications and citations from a
> single author in reasonable time.
> Do you have a recommendation for which proxy solution to use to get
> reliable results?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/scholarly-python-package/scholarly/discussions/263>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AABFH4NIU2SLY7L6KTRNNNDTAC3X3ANCNFSM4X6XIKDQ>
> .
>
-- 
Sent from Mobile
